pupper_install.sh

```diff
@@ -62,7 +62,6 @@ touch mini_pupper_ros/mini_pupper_navigation/AMENT_IGNORE
 cd ~/ros2_ws
 rosdep install --from-paths src --ignore-src -r -y --skip-keys=joint_state_publisher_gui --skip-keys=rviz2 --skip-keys=gazebo_plugins --skip-keys=velodyne_gazebo_plugins
 sudo apt install ros-humble-teleop-twist-keyboard
-sudo apt install ros-humble-teleop-twist-joy
```

Comment:
this was added in the following pr by Joey
https://github.com/mangdangroboticsclub/mini_pupper_ros/pull/98

I saw there is a bunch of things added in the README, control robot with joystick. Not sure if we shall remove this line here.